src/test/java/Problem1Test.java

```diff
@@ -42,14 +42,14 @@ public void testInorderTraversal() {
     }
 
     @Test
-    public void testLevelOrderTraversal() {
+     public void testLevelOrderTraversal() {
         List<LevelOrderTestCase<Integer>> testCases = getLevelOrderTraverseTestCases();
         for (int i = 0; i < testCases.size(); i++) {
             LevelOrderTestCase<Integer> testCase = testCases.get(i);
             List<List<Integer>> actual = LevelOrderTraverse.levelOrder(testCase.tree);
-            assertEquals(testCase.expect.toString(), actual.toString());
+            //assertEquals(testCase.expect.toString(), actual.toString());
```

Comment:
big no no

Reply:
never ever comment out tests (okay for the extra credit problem when it's specified through other communication)